#!/usr/bin/python3
def raise_exception():
    print("{:d}".format((1,)))


if __name__ == "__raise_exception__":
    raise_exception()
